### DISCUSSION TITLE
cname > domian hosted on nginx proxy manager

### DISCUSSION THREAD

**Discussion by redimongo** [NginxProxyManager/nginx-proxy-manager#2731] [Q&A]
I am trying to get cname forwarders working. 

Basically we offer a white label portal to customers. 

We need them to place portal.example.com as a cname 

For example say their domain was portal.mypod.com 

That would cname to portal.example.com

The nginx-proxy-manager would just forward that onto the docker container in this case it's called NextJS:3000

Currently am getting the following page. 

![image](https://user-images.githubusercontent.com/8399907/226875207-2e97ba49-d70c-4bfb-b0d6-68ddd2b3688d.png)

Any help welcomed.

**Comment by redimongo** [comment]
Solution to my problem was to go into advance tab and type in the following:

```
server_name ~.;
```

Now the issue is that all style sheets and every base URL will go to whatever server_name was picked.

Could this be a problem long term - maybe. I am still to test how this works with nextjs and next-auth.

**Reply by dniald** [reply]
did this really solved your problem by updating server_name in the site.conf?
